### PR TITLE
Version bump + fixes + change logs

### LINKED ISSUE #72: Possible for `sellall except` and `sellall slot` command?
Would it be possible to make a `sellall except item` and a `sellall slot` command 
`sellall except item` so we could keep one item (like the miniboss weapon) in our backpacks without having to sell items one by one? Can get quite consuming when looting a bunch of chests

`sellall slot` so we could just sell a particular item slot in our backpack? 

Not sure how much work these would entail but thank you for considering! :)

### LINKED ISSUE #97: [Change Log] Some spelling errors
Some have been changed already. Just adding this to be seen.

https://ptb.discordapp.com/channels/240154543684321280/240212783503900673/674739667567902741

### LINKED ISSUE #109: Allow backpack to specify certain slots
Allow `[p]backpack` to specify certain slots for easier navigation of a lot of equipment.

For example `[p]backpack head` will list only items that can be equipped in the head slot.

### LINKED ISSUE #115: Allow duplicate set items to be sold
I've already discussed this with **Draper** and some adventurers like me and ended up thinking that duplicate set items should be sell-able instead of just populating the qty in the backpack since set items can only be equipped or stay in the backpack. 

tl;dr - draper told me to file this issue because he always forget it.

### LINKED ISSUE #124: [Suggestion] [p] backpack <rarity> command
Since backpack sellall <rarity> works well, I think it would be also great if bot can also show the item list by rarity.

### LINKED ISSUE #138: Bard Skill [p]Music does not do anything.
so with music - KTCAOP :game_die:(10) + :musical_note: 34 + :speech_left:97
Without Music - KTCAOP :game_die:(10) + :musical_note: 33 + :speech_left:97

With Music - KTCAOP :game_die:(11) + :musical_note: 31 + :speech_left:92
Without Music - KTCAOP :game_die:(16) + :musical_note: 33 + :speech_left:92

watdakwan and Darius can confirm.

### PR DESCRIPTION
**Pull request by Drapersniper**
Fixes #124 
Fixes #115
Fixes #97
Fixes #109
Partially addresses #72
Fixes #138

### CODE REVIEW

File: adventure/adventure.py [hunk 1]
```diff
@@ -792,24 +827,103 @@ def add_stats(word_stats):
...
+                item = character.backpack[equip_item.name]
+            except KeyError:
+                return
+
+            if item.rarity != "set":
+                return await smart_embed(ctx, _("You can only disassemble set items."))
+            if character.heroclass["name"] != "Tinkerer":
+                roll = random.randint(0, 1)
+            else:
+                roll = random.randint(0, 5)
```
Comment (Drapersniper) [inline adventure/adventure.py]:
```suggestion
                roll = random.randint(0, 3)
```
